packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/updateModalCancelIconButton.js

```diff
@@ -12,7 +12,7 @@ const UpdateModalCancelIconButton = props => {
     const { handleClick } = useResetForm({ onClick });
 
     return (
-        <button className={className} onClick={handleClick}>
+        <button className={className} onClick={handleClick} type="button">
```

Comment:
So important! The default `type` is `submit`, and we listen for the form's `submit` event rather than just the submit button's `click` event. 😁